tools/serverpod_cli/lib/src/runner/serverpod_command_runner.dart

```diff
@@ -160,6 +160,12 @@ class ServerpodCommandRunner extends CommandRunner {
   ArgParser get argParser => _argParser;
   final ArgParser _argParser = ArgParser(usageLineLength: log.wrapTextColumn);
 
+  void addCommands(List<Command> commands) {
+    for (var command in commands) {
+      addCommand(command);
+    }
```

Comment:
Question: Where is this addCommand coming from? And can we not simply do an add all on the underlying list instead of looping?

Reply:
Add command is part of the CommandRunner interface and unfortunately it doesn't support adding a list. That is why I added this method, so what we could supply a list instead of adding each command individually.

This allows us to add if statements when building the list before adding the commands.